osh/worker/csmock_runner.py

```diff
@@ -1,32 +1,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 # SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
 
-"""
-
-csmock python api
-
-{
-    "scan":
-    {
-        "analyzer-version-clang": "3.4",
-        "analyzer-version-cppcheck": "1.66",
-        "analyzer-version-gcc": "4.9.1",
-        "exit-code": 0,
-        "host": "quahog",
-        "mock-config": "fedora-21-x86_64",
-        "store-results-to": "/tmp/asd",
-        "time-created": "2014-09-01 18:30:19",
-        "time-finished": "2014-09-01 18:38:15",
-        "tool": "csmock",
-        "tool-args": "'/bin/csmock' '-t' 'cppcheck,gcc,clang' '--no-scan' '-r' 'fedora-21-x86_64' '-o' 'asd' '--force'",
-        "tool-version": "csmock-1.3.2.20140829.165742.ge16c941-1.fc21"
-    },
-    "defects": ""
-}
-"""
```

Comment:
I think we still need somewhere to be able to know what's inside the json output, if inline doc is not preferred, an external link(i.e., point to some data declaration from the csmock repo) would be helpful too.

Unfortunately the repo doc doesn't tell: https://github.com/csutils/csmock/tree/main/doc

Reply:
Yes, there is no such documentation in `csmock` yet.

Reply:
I'll update the docstrings with the current format of the JSON file with results.

Reply:
Addressed in  824beeba205847a66681f95503d7f457f77689de.